pep-0648.rst

```diff
@@ -70,7 +70,7 @@ though, and it is not the main driver of this change, but should the change
 happen, it will also improve the situation for these users, as rather than
 having a `sitecustomize.py` which performs all those actions, they can have
```

Comment:
Single backticks are code in Markdown, but double are needed for RST.

Here and elsewhere:
```suggestion
having a ``sitecustomize.py`` which performs all those actions, they can have
```

(In the future, this would be caught by linting in https://github.com/python/peps/pull/1668)